### PR TITLE
Format the codes of Quiz question(4th) in section Bronze/Complete Search with Recursion

### PR DESCRIPTION
**Pull request by KunalSin9h**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which inclues the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

**Review by SansPapyrus683 — APPROVED** [review]
sure?
best for another to review before i merge tho

**Review by ryanchou-dev — APPROVED** [review]
should be ok

File: content/2_Bronze/Complete_Rec.mdx [hunk 1]
```diff
@@ -1234,74 +1234,74 @@ make code consistent, improve some of these explanations
 	    int A[] = {1, 2, 3, 4, 5};
 
 	    void generateSubsets(int i, vector<int> &subset) {
```
Comment (bqi343) [inline content/2_Bronze/Complete_Rec.mdx:1236]: ^ there are still spaces before and after the `generateSubsets` function

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Review by bqi343 — CHANGES_REQUESTED** [review]
not sure why review was re-requested despite no changes having been made

**Review by bqi343 — APPROVED** [review]

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**